lucet-idl/lucet-idl-test/resources/rust_host/src/harness.rs

```diff
@@ -1,6 +1,281 @@
-// PLACEHOLDER FILE
-// this file is overwritten, and restored, by lucet-idl-test
-pub struct Ctx;
-pub fn ctx() -> Box<Ctx> {
-    Box::new(Ctx)
+use crate::idl::spec::*;
```

Comment:
changes to this file shouldnt be checked in - unfortunately its a side effect of exiting lucet-idl-test via ctrl-c. unfortunately, we need to just leave a placeholder file checked in.

Reply:
Ah! Thanks for confirming :slightly_smiling_face: